corporate/lib/stripe.py

```diff
@@ -4832,6 +4858,26 @@ def invoice_plans_as_needed(event_time: Optional[datetime] = None) -> None:
                     plan.invoice_overdue_email_sent = True
                     plan.save(update_fields=["invoice_overdue_email_sent"])
                 continue
```

Comment:
Hmm, I think this `continue/elif` construction creates risk that if a server isn't sending audit log updates, which seems possible for some cases of a fixed-price plan, then we might not run the new `elif` block.

One theory is that the new `elif` block should just be an independent `if` block placed before this one.

Reply:
That's right. 

> the new elif block should just be an independent if block placed before this one.

Made the changes.